web/src/scroll_bar.ts

```diff
@@ -41,14 +10,6 @@ export function set_layout_width(): void {
     }
 }
```

Comment:
As a follow-up, we could probably look at changing this to involve a CSS class on body that we toggle, making this file no longer have any `.css` calls.